README.md

```diff
@@ -4,7 +4,7 @@
 ## Meeting Time
 
 CNCF's Telecom User Group meets on the first Monday of the month. The meeting time switches monthly between:
-- 15:00 UTC (8am Pacific Daylight Time)
+- 15:00 UTC (7am Pacific Standard Time)
```

Comment:
This will be valid until November 7th, then it will need to be updated.  I'm good with changing it for a week though 🤷🏻